libraries/lib-datahandler/src/main/java/com/siemens/sw360/datahandler/thrift/ThriftClients.java

```diff
@@ -125,4 +126,7 @@ private static TProtocol makeProtocol(String url, String service) {
         return new VulnerabilityService.Client(makeProtocol(BACKEND_URL, VULNERABILITY_SERVICE_URL));
     }
 
+    public LicenseInfoService.Client makeLicenseInfoClient() {
+        return new LicenseInfoService.Client(makeProtocol(BACKEND_URL, PROJECT_SERVICE_URL));
```

Comment:
It took me quite a while to figure out why everything builds fine, but `LicenseInfoServiceClient` can't find method `getLicenseInfoFileForProject` in runtime.
You may want to correct that `*SERVICE_URL` argument.